data/json/items/migration.json

```diff
@@ -1,5 +1,10 @@
 [
   {
+    "id": "UPS_on",
```

Comment:
Looks like `jq` sorts by Unicode value by default, meaning capital letters go first.

Fortunately it's just one entry, I'll fix that manually.